Tests/TuistKitTests/Services/DocServiceTests.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
+import TuistSupport
+import XCTest
+import TSCBasic
+import TuistCore
+
+@testable import TuistKit
+@testable import TuistSupportTesting
+@testable import TuistDocTesting
+@testable import TuistCoreTesting
+
+final class TuistDocServiceTests: TuistUnitTestCase {
+    var subject: DocService!
+    
+    var projectGenerator: MockProjectGenerator!
+    var swiftDocController: MockSwiftDocController!
+    var opener: MockOpener!
+    
+    override func setUp() {
+        super.setUp()
+        
+        projectGenerator = MockProjectGenerator()
+        swiftDocController = MockSwiftDocController()
+        opener = MockOpener()
+        fileHandler = MockFileHandler(temporaryDirectory: { try self.temporaryPath() })
+        
+        subject = DocService(projectGenerator: projectGenerator,
+                             swiftDocController: swiftDocController,
+                             opener: opener,
+                             fileHandler: fileHandler)
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        
+        subject = nil
```

Comment:
I think it's safer to nullify the mock properties, too

Reply:
c98b78b4